plugins/wysiwygarea/plugin.js

```diff
@@ -93,9 +98,82 @@
 						return;
 					}
 
-					editor.editable( new framedWysiwyg( editor, iframe.$.contentWindow.document.body ) );
+					editor.editable( new framedWysiwyg( editor, iframe.getFrameDocument().getBody() ) );
 					editor.setData( editor.getData( 1 ), callback );
+
+					var isIE11 = CKEDITOR.env.ie && CKEDITOR.env.version === 11;
+
+					// (#4462)
+					// Skip IE's below version 11. They don't support MutationObserver.
+					if ( !CKEDITOR.env.ie || isIE11 ) {
+						if ( isIE11 ) {
+							editor.on( 'mode', attachIframeReloader, { iframe: iframe, editor: editor, callback: callback } );
+						}
+
+						observeEditor();
+					}
```

Comment:
We could flatten it even more:
```suggestion
					if ( CKEDITOR.env.ie && !isIE11 ) {
						return;
					}
					
					if ( isIE11 ) {
						editor.on( 'mode', attachIframeReloader, { iframe: iframe, editor: editor, callback: callback } );
					}

					observeEditor();
```

But the conditions now are quite clear so we can also leave it as is if you want 🤔

Reply:
Well, it looks even better - so yes - lets make it flat :D